packages/twenty-server/src/modules/calendar/calendar-event-import-manager/drivers/google-calendar/services/google-calendar-get-events.service.ts

```diff
@@ -90,4 +77,34 @@ export class GoogleCalendarGetEventsService {
       nextSyncCursor: nextSyncToken || '',
     };
   }
+
+  private handleError(error: GaxiosError) {
```

Comment:
We can improve the naming here as handleError is actually a throwCalendarEventError